stress/frontend/config/stressTestInfra.conf

```diff
@@ -0,0 +1,3 @@
+sweetSpot = 14 //maximum throughput the backend can handle
```

Comment:
it corresponds to the number of Responder Actors (each throttled to the minRate) the backend uses to consume work. will make it clearer in the comment (and will change the comments to #)